app/uk/gov/hmrc/tariffclassificationfrontend/views/forms/components/input_radiogroup.scala.html

```diff
@@ -22,13 +22,14 @@
         inputClass: Option[String] = None,
         hint: Option[String] = None,
         labelClass: Option[String] = None,
+        heading:String = "Review status",
```

Comment:
Do you mind if we pull the heading out of this class. Seems a bit weird to have a default value of "Review Status" on a fairly generic component. 
Admittedly I think I made this so I should have spotted it originally...

Reply:
Do you mean pull it out completely (so the component doesn't render a heading) or just remove the default value?  I agree that a "review" specific default is daft

Reply:
Pull it out completely, I think there would be times we would want radio groups but not want the heading